tla/consensus/MCccfraftAtomicReconfig.cfg

```diff
@@ -75,8 +75,8 @@ INVARIANTS
     TypeInv
     MonoTermInv
     MonoLogInv
-    LogConfigurationConsistentInv
-    NoLeaderInTermZeroInv
+    \* LogConfigurationConsistentInv
```

Comment:
I think just remove if not wanted instead of commenting out. It should be case that `LogConfigurationConsistentInv` can be fixed if we want to keep it